datatype/_target.lua

```diff
@@ -2,7 +2,7 @@
 --- @field public Aego cachedbuff|fun(): string Returns the name of the Aego spell if the Target has one
 --- @field public AggroHolder spawn|fun(): spawn Returns the target's current target.
 --- @field public Beneficial cachedbuff|fun(): string Returns the name of the Beneficial spell if the Target has one. This will skip "player" casted buffs, but will show NPC Casted buffs and some AA buffs.
---- @field public Buff cachedbuff|fun(): cachedbuff Returns the target's spell by buff index number or the first buff if no Index is provided.
+--- @field public Buff cachedbuff|fun(name: string|integer): cachedbuff Returns the target's spell by buff index number or the first buff if no Index is provided.
```

Comment:
The name field needs to be optional
```suggestion
--- @field public Buff cachedbuff|fun(name?: string|integer): cachedbuff Returns the target's spell by buff index number or the first buff if no Index is provided.
```

Reply:
Thanks, I missed that part!